components/AboutSite.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="AboutSite">
+    <div class="AboutSite-heading">
+      <h3 class="AboutSite-title">{{ $t('当サイトについて') }}</h3>
+    </div>
+    <div class="AboutSite-body">
+      <i18n
+        tag="p"
+        path="当サイトは新型コロナウイルス感染症（COVID-19）に関する最新情報を提供するために、{volunteer}が開設したまとめサイトです。"
+      >
+        <template v-slot:volunteer>{{ $t('有志の仲間') }}</template>
+      </i18n>
+      <div class="AboutSite-Button">
+        <nuxt-link class="link" to="/about">`·$t('詳しくはこちら')`</nuxt-link>
```

Comment:
```suggestion
        <nuxt-link class="link" to="/about">
          {{ $t('詳しくはこちら') }}
        </nuxt-link>
```
eslintを回避するにはこのようにしてください。

Reply:
ありがとうございます